qemu/tests/migration_with_vsock.py

```diff
@@ -61,19 +61,27 @@ def ping_pong_migration(repeat_times):
             )
 
     def input_character_vsock():
-        connected_str = r"Connection from cid*"
-        nc_vsock_listen(nc_vsock_bin, port, session)
-        host_vsock_session = nc_vsock_connect(nc_vsock_bin, guest_cid, port)
+        if vsock_test_tool == "ncat":
+            vsock_listen(tool_bin, port, session)
+            host_vsock_session = vsock_connect(tool_bin, guest_cid, port)
         send_data = "Hello world"
-        check_received_data(test, session, connected_str)
+        if vsock_test_tool == "nc_vsock":
+            vsock_listen(tool_bin, port, session)
+            host_vsock_session = vsock_connect(tool_bin, guest_cid, port)
+            connected_str = r"Connection from cid*"
+            check_received_data(test, session, connected_str)
         error_context.context('Input "Hello world" to vsock.', logging.info)
         host_vsock_session.sendline(send_data)
         check_received_data(test, session, send_data)
         return host_vsock_session
 
+    vsock_test_tool = params["vsock_test_tool"]
     vm = env.get_vm(params["main_vm"])
     session = vm.wait_for_login()
-    nc_vsock_bin = compile_nc_vsock(test, vm, session)
+    if vsock_test_tool == "nc_vsock":
+        tool_bin = compile_nc_vsock(test, vm, session)
+    if vsock_test_tool == "ncat":
+        tool_bin = "/usr/bin/ncat"
```

Comment:
Suggest to use `path.find_command("ncat")` to get the real path.

Reply:
Updated.